### DISCUSSION TITLE
Footer icons Font Awesome not working properly

### DISCUSSION THREAD

**Discussion by ewuerger** [pradyunsg/furo#590] [Potential Issue] [answered]
Hi and thank you for this awesome theme,

I've set up a GitHub footer icon from Font Awesome by following the [documentation](https://pradyunsg.me/furo/customisation/footer/#font-awesome), but it just wouldn't display. From the [Font Awesome website](https://fontawesome.com/icons/github?s=solid&f=brands) it displays this code snippet:
``` html
<i class="fa-brands fa-github"></i>
```

After adding the missing `fa-brands` class it was working. This may also fix #400 and I suggest to update the docs section.

**Comment by umarcor** [comment, marked as answer]
Done in #629.
